components/logins/src/error.rs

```diff
@@ -181,6 +181,24 @@ impl GetErrorHandling for Error {
                 })
                 .report_error("logins-sync"),
             },
+            Error::SqlError(rusqlite::Error::SqliteFailure(err, _)) => match err.code {
+                rusqlite::ErrorCode::DatabaseCorrupt => {
+                    ErrorHandling::convert(LoginsApiError::UnexpectedLoginsApiError {
+                        reason: self.to_string(),
+                    })
+                    .report_error("logins-db-corrupt")
+                }
+                rusqlite::ErrorCode::DiskFull => {
+                    ErrorHandling::convert(LoginsApiError::UnexpectedLoginsApiError {
+                        reason: self.to_string(),
+                    })
+                    .report_error("logins-db")
```

Comment:
maybe `logins-db-disk-full` here? Totally up to you but would seem clearer without context on the sentry side.

Reply:
Oh yeah, that's much better.  I think I just forgot to type out the entire string.